### PR TITLE
Forward rel/7.0 to rel/7.1

### PR DESCRIPTION
**Pull request by alexanderchan-scality**
Forward from rel/7.0 to rel/7.1

### CODE REVIEW

**Comment by nicolas2bert** [comment]
@ironman-machine r+

File: tests/functional/aws-node-sdk/test/multipleBackend/objectTagging.js [hunk 1]
```diff
@@ -154,6 +154,9 @@ function mpuWaterfall(params, cb) {
 
 describeSkipIfNotMultiple('Object tagging with multiple backends',
 function testSuite() {
+    if (!process.env.S3_END_TO_END) {
```
Comment (LaurenSpiegel) [inline tests/functional/aws-node-sdk/test/multipleBackend/objectTagging.js]: why do we need this condition?

Comment (alexanderchan-scality) [reply tests/functional/aws-node-sdk/test/multipleBackend/objectTagging.js]: I had followed the form of what has been added to master, such that retries should not be allowed when the var is set. I see that it won't be used in rel/6.3, I will make the proper changes.

**Comment by nicolas2bert** [comment]
@alexanderchan-scality, just wondering why has it never been merged?

**Pull request closed without merging**